app/controllers/comment_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentController < ApplicationController
+  def create
+    @comment = Comment.new(author_id_id: params[:user_id], post_id_id: params[:post_id],
+                           text: params[:comment][:text])
```

Comment:
- [ OPTIONAL ] I highly recommend to add `comment_params` mathod since it is considered as [strong parameters](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) so kindly check this link how to add strong parameters ✅✨🚀💯